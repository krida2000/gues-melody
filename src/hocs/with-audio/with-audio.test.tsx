import * as React from "react";
import * as renderer from "react-test-renderer";
import withAudio from "./with-audio";

class MockComponent extends React.PureComponent {
  render() {
    const {children} = this.props;
    return <div>{children}</div>;
  }
}

const MockComponentWithAudio = withAudio(MockComponent);

it(`With audio correctly render`, () => {
  const tree = renderer
    .create(
      <MockComponentWithAudio src={``} playButtonClickHandler={jest.fn} isPlaying={false}/>, {
        createNodeMock() {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
