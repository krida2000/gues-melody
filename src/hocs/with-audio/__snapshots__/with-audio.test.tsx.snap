// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With audio correctly render 1`] = `
<div>
  <audio />
</div>
`;
